Cargo.toml

```diff
@@ -40,5 +40,18 @@ criterion = { version = "0.5.1", default-features = false, features = ["cargo_be
 lazy_static = "1.4.0"
 rand = "0.8.5"
 
+# Compilation settings for performance
+[profile.release]
+panic = "abort"
```

Comment:
Not sure whether it is worth losing backtraces for I suppose no gains in non-panic executions?

Reply:
Debug and backtrace info increases the size of the code, so it can have an impact on performance. Particularly in recursive code.

For example: `rec_karatsuba_mul()` slows down by 25x with debug info and overflow checks on. (I don't know how much of that is the debug info or the overflow checks.)

See the last hidden benchmarks in:
https://github.com/Inversed-Tech/eyelid/pull/32#issuecomment-2054782381